python/nav/web/crispyforms.py

```diff
@@ -195,3 +195,16 @@ class and template definitions.
         form_id=form_id,
         form_class=form_class,
     )
+
+
+class FormDiv:
+    """A class representing a div in a form layout.
+
+    :param fields: A list of fields to include in the div.
+    :param css_classes: Additional CSS classes to apply to the div.
+    """
+
+    def __init__(self, fields: Optional[list] = None, css_classes: str = ''):
```

Comment:
```suggestion
    def __init__(self, fields: Optional[list] = None, css_classes: Optional[str] = ''):
```

if I understand correctly that this technically might also be empty?

Reply:
Might be empty but not sure it can be `None`. For consistency I think it should stay the same as the other classes.

Reply:
https://github.com/Uninett/nav/blob/31de3bf2d56056b08fdb6a0e64697d169fe5856a/python/nav/web/templates/custom_crispy_templates/form_div.html#L4

Since we here just check for `if field.css_classes` it should not be a problem for it to be empty

Reply:
yous right, but I still think it should be consistent with `FormRow` and `FormColumn`

Reply:
It does not make a functional difference, but I think your suggestion is cleaner

Reply:
I had a look at how `FormRow` is rendered and I am considering if it makes sense to make the HTML template more similar to this one and also to make `css_classes` optional, any opinions?

Reply:
I think that can make sense, using the Optional tag and allowing None makes sense for an argument that is effectively optional, feels cleaner than defaulting to empty string

Reply:
Alright, I can create a PR there

Reply:
#3187